upgrading/automated_upgrades.adoc

```diff
@@ -547,12 +547,12 @@ catalog and service brokers is not supported.
 To upgrade to these features:
```

Comment:
Otherwise, the next step depends on what additional features you have previously enabled.

Reply:
Fixt